crm_create_project/views/crm_lead.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="crm_lead_view_form_crm_project_project" model="ir.ui.view">
+        <field name="name">CRM Lead Form Create Project</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="header" position="inside">
+                <button
+                    string="Create Project"
+                    name="%(crm_create_project.crm_create_project_action)d"
+                    type="action"
+                    class="oe_highlight"
+                    context="{'default_project_name': name, 'default_lead_id': id}"
+                    attrs="{'invisible': ['|', ('active','=',False), ('project_id', '!=', False)]}"
+                />
+            </xpath>
+            <xpath expr="//group[@name='Misc']" position="inside">
+                <field name="project_id" readonly="1" />
```

Comment:
question: why readonly? What if the project is created by somebody else and you just want to link it here?

Reply:
This is not a glue module linking the crm_lead and project_project models. The field is for the project created, if it could be updated it would not be the one created from the lead/opportunity.
Maybe it could be interesting to change the name of the field to project created, what do you think?

Reply:
I understand the use case, but IMHO it might not reflect the reality.

For example, imagine a customer wants a very specific project. Salesperson doesn't have permission to create a custom project, so he calls the project manager and asks him to create it. Once done, salesperson comes back to the lead to link that project, but it's not possible.

As long as we cover the use case and the field's purpose is well explained, I see no strong reason to forbid users to use it differently. Odoo isn't in general so strict.

Reply:
Please @EmilioPascual add this discussions to ROADMAP o KNOWISSUES please and go ahead @yajo . I guess with use the product evolves, we cannot cover all use cases all the time. The most importat is evolve! OK? 😄 ❤️ What do you think?

Reply:
I was thinking about that, maybe @yajo is right. I'm going to try with no readonly field, it may be good for planning extensions to previously created projects.